meinberlin/assets/scss/components/_filter_bar.scss

```diff
@@ -144,6 +144,11 @@ $divider-margin: 1.25rem;
         padding-left: 0;
     }
 
+    .btn-primary {
```

Comment:
Don't we have a btn--primary normally? This seems to be quite special to something? Maybe we need to add another class and not use that one?!